.github/workflows/pkgci_regression_test.yml

```diff
@@ -237,61 +235,33 @@ jobs:
             --durations=0 \
             --config-files=${MODELS_CONFIG_FILE_PATH}
 
-      - name: Run external tests - models with real weights and additional flags
-        if: "matrix.models-extra-flags-config-file != '' && !cancelled()"
+      - name: "Running SDXL special model tests"
+        if: ${{ !cancelled() }}
         run: |
           source ${VENV_DIR}/bin/activate
-          pytest SHARK-TestSuite/iree_tests/pytorch/models \
+          pytest ./experimental/regression_suite/shark-test-suite-models/sdxl \
+            -k ${{ matrix.backend }} \
```

Comment:
The `amdgpu_vulkan` matrix entry doesn't specify `backend`, so this is failing with `pytest: error: argument -k: expected one argument`

https://github.com/iree-org/iree/actions/runs/9779308546/job/27006098126?pr=17803#step:11:28